src/mongocxx/options/encrypt.cpp

```diff
@@ -51,24 +49,35 @@ const stdx::optional<encrypt::encryption_algorithm>& encrypt::algorithm() const
     return _algorithm;
 }
 
+const stdx::optional<bsoncxx::types::bson_value::view_or_value>& encrypt::key_id() const {
+    return _key_id;
+}
+
 void* encrypt::convert() const {
     using libbson::scoped_bson_t;
 
     mongoc_client_encryption_encrypt_opts_t* opts = libmongoc::client_encryption_encrypt_opts_new();
 
-    if (_key_id_set && _key_alt_name) {
+    if (_key_id && _key_alt_name) {
```

Comment:
Do you recall if this empty if block was intentional? If it's just for the purpose of documentation I'd suggest removing the if and expanding the comment to say "libmongoc will error if both id and alt name are set".

Reply:
I think it was just a readability thing, happy to remove it.